docs/developer-docs/setup/hello-world.mdx

```diff
@@ -0,0 +1,254 @@
+# Hello, world!
+
+This guide shows how to get started developing on ICP quickly by deploying your first 'Hello, world!' smart contract. 
+
+## Install dependencies
+
+### Install the IC SDK (v0.15.2 and newer)
+
+```bash
+sh -ci "$(curl -fsSL https://internetcomputer.org/install.sh)"
+```
+
+### Install Node.js
+
+```bash
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
+nvm install 18
+```
+
+## Start the local replica
+
+```bash
+dfx start
+```
+
+## Create a new project
+
+<Tabs groupId="language">
+<TabItem value="motoko" label="Motoko" default>
+
+```bash
+dfx new hello
+cd hello
+npm install
+```
+
+</TabItem>
+<TabItem value="rust" label="Rust">
+
+```bash
+dfx new hello --type=rust
+cd hello
+npm install
+```
+
+</TabItem>
+
+<TabItem value="typescript" label="TypeScript">
+
+```bash
+npx azle new azle_hello
+cd hello
+npm install
+```
+
+</TabItem>
+
+<TabItem value="python" label="Python">
+
+Create the project directory and file structure:
+
+```bash
+mkdir kybra_hello
+cd kybra_hello
+
+mkdir src
+
+touch src/main.py
+touch dfx.json
+```
+
+Create a virtual environment:
+
+```bash
+~/.pyenv/versions/3.10.7/bin/python -m venv venv
```

Comment:
does `venv` need to be installed, or is that typically included with the OS?

Reply:
From my understanding it needs to be installed since it is a Python-specific package.